src/components/FAQ.js

```diff
@@ -0,0 +1,129 @@
+          <h2 className="accordion-header" id="headingFive">
+            <button className="accordion-button text-dark bg-light" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
+             05. Can I pay in Bitcoin?
+            </button>
+          </h2>
+          <div id="collapseFive" className="accordion-collapse collapse" aria-labelledby="headingFive" data-bs-parent="#accordionExample">
+            <div className="accordion-body">
+                <p>Unfortunately, we do not accept Bitcoin at this time.</p>
+            </div>
+          </div>
+        </div>
+        <div className="accordion-item">
+          <h2 className="accordion-header" id="headingSix">
+            <button className="accordion-button text-dark bg-light" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
+             06.Do you offer gluten-free pizza?
+            </button>
+          </h2>
+          <div id="collapseSix" className="accordion-collapse collapse" aria-labelledby="headingSix" data-bs-parent="#accordionExample">
+            <div className="accordion-body">
+                <p>Yes, we offer a variety of gluten-free and vegan pizzas. Just let us know of your dietary restrictions when you order. </p>
+            </div>
+          </div>
+        </div>
+        <div className="accordion-item">
+          <h2 className="accordion-header" id="headingSeven">
+            <button className="accordion-button text-dark bg-light" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSeven" aria-expanded="false" aria-controls="collapseSeven">
+             07. Do I need to create an account to make purchases?
+            </button>
+          </h2>
+          <div id="collapseSeven" className="accordion-collapse collapse" aria-labelledby="headingSeven" data-bs-parent="#accordionExample">
+            <div className="accordion-body">
+                <p>Creating an account is optional. You can checkout as a guest if preferred. Registered accounts can view order history and save shipping addresses. </p>
+            </div>
+          </div>
+        </div>
+        <div className="accordion-item">
+          <h2 className="accordion-header" id="headingEight">
+            <button className="accordion-button text-dark bg-light" type="button" data-bs-toggle="collapse" data-bs-target="#collapseEight" aria-expanded="false" aria-controls="collapseEight">
+             08. What payment methods do you accept?
+            </button>
+          </h2>
+          <div id="collapseEight" className="accordion-collapse collapse" aria-labelledby="headingEight" data-bs-parent="#accordionExample">
+            <div className="accordion-body">
+                <p>We accept Visa, Mastercard, American Express, Discover, PayPal and Amazon Pay. Payment is processed at the time the order is placed. </p>
```

Comment:
I think we should remove Paypal and Amazon Pay here. We'll shoot for trying to get cc set up.

Reply:
done

Reply:
👍